ydb/core/blobstorage/dsproxy/dsproxy_request_reporting.cpp

```diff
@@ -0,0 +1,46 @@
+#include "dsproxy_request_reporting.h"
+
+namespace NKikimr {
+
+static std::array<std::atomic<bool>, 7> ReportPermissions;
+
+bool AllowToReport(NKikimrBlobStorage::EPutHandleClass handleClass) {
+    return ReportPermissions[(ui32)handleClass - 1].exchange(false);
```

Comment:
please don't do that, I'm begging you

what if in the next year I'll add new EPutHandleClass? All that system will break